modules/ROOT/pages/index.adoc

```diff
@@ -67,22 +67,22 @@ The latest ownCloud Desktop Sync Client release, suitable for production use.
 These are the older ownCloud releases.
 Users of these releases are *strongly encouraged* to upgrade to the latest production release.
 
+=== ownCloud 9.1
+
+* https://doc.owncloud.org/server/9.1/admin_manual/[Administration Manual]
+  (https://doc.owncloud.org/server/9.1/ownCloud_Administration_Manual.pdf[Download PDF])
+* https://doc.owncloud.org/server/9.1/developer_manual/[Developer Manual]
+  (https://doc.owncloud.org/server/9.1/ownCloud_Developer_Manual.pdf[Download PDF])
```

Comment:
Please see issue: https://github.com/owncloud/docs/issues/467 (Broken links in docs production)